openwisp_radius/api/views.py

```diff
@@ -326,13 +328,35 @@ def get_queryset(self):
 accounting = AccountingView.as_view()
 
 
-class BatchView(TokenAuthorizationMixin, generics.CreateAPIView):
-    authentication_classes = (TokenAuthentication,)
+class DispatchOrgMixin(object):
+    def dispatch(self, *args, **kwargs):
+        try:
+            self.organization = Organization.objects.get(slug=kwargs['slug'])
+        except Organization.DoesNotExist:
+            raise Http404()
+        return super().dispatch(*args, **kwargs)
+
+    def validate_membership(self, user):
```

Comment:
I'm not sure if this is the right moment to do it, but take a look at https://github.com/openwisp/openwisp-users/issues/150
It would be great to solve that issue and create a general way for the API views of OpenWISP to do this kind of check.